helpers/dmesg.py

```diff
@@ -37,15 +38,11 @@ class DmesgOopsFinder(object):
 
     def __init__(self):
         self.node = remote.tempesta
+        self.start_time = time.time()
```

Comment:
Tempesta can be started on other vm, not the same that was used for framework itself. So time between systems may differ, and we need to get current time value on Tempesta node.

Reply:
Changed to `date +%s.%N` running on the remote host.